README.md

```diff
@@ -1,2 +1,159 @@
-# juttle-ga-adapter
-juttle adapter for google analytics
+# Juttle Google Analytics Adapter
+
+[![Build Status](https://travis-ci.org/juttle/juttle-ga-adapter.svg?branch=master)](https://travis-ci.org/juttle/juttle-ga-adapter)
+
+This is an adapter that allows Juttle to read data from [Google Analytics](http://www.google.com/analytics/) using the [Core Reporting API](https://developers.google.com/analytics/devguides/reporting/core/v3/)
+
+It can pull data from various web properties and views to get visibility into usage data across sites.
+
+## Examples
+
+### Count pageviews and sessions for each web property over the past week
+
+```juttle
+read ga -last :day: | reduce pageviews=sum(pageviews), sessions=sum(sessions) by webProperty
+```
+
+```
+┌──────────────┬────────────┬─────────────────────────────────────────┐
```

Comment:
This table is pretty ghetto, how about something like https://github.com/juttle/juttle-elastic-adapter/pull/98/files#diff-04c6e90faac2675aa89e2176d2eec7d8R123?